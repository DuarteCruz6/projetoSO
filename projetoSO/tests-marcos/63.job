WRITE [(b8,95)(l2,68)(p1,39)]
WRITE [(l6,52)(k4,53)]
WRITE [(c3,33)(j2,39)(y4,67)(l1,53)]
WRITE [(l4,78)(h7,35)(b0,8)(e9,14)(l8,30)]
WRITE [(q6,41)(e6,46)(g8,7)(s0,66)]
WRITE [(e0,59)(l7,68)]
DELETE [l4,e6,q6,e0]
SHOW
BACKUP
BACKUP
WRITE [(e3,91)(d3,38)(d6,3)]
WRITE [(y6,94)(e2,79)(e7,65)(h3,12)]
DELETE [h7,e7,b0,d6,h3]
