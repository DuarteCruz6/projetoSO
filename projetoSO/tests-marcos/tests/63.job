WRITE [(g6,10)(i1,10)(u8,32)(o1,2)]
WRITE [(k7,23)(z5,34)(a9,32)]
SHOW
DELETE [o1,g6,u8,k7,z5]
DELETE [i1,a9]
WRITE [(p6,63)(v2,23)(j2,70)(n9,57)(l5,54)]
WRITE [(g3,74)(a2,33)(w5,87)]
DELETE [v2,p6,j2,n9]
WRITE [(l3,9)(w8,93)(i8,74)(l9,81)]
BACKUP
BACKUP
WRITE [(u7,38)(y1,5)(o9,37)]
WRITE [(r9,28)(i5,1)(i0,67)(l1,84)(r2,50)]
READ [i8]
WRITE [(p4,91)(b5,71)(a1,86)(y9,84)(u5,42)]
WRITE [(u9,4)]
SHOW
READ [b5]
WAIT 3000
WRITE [(k0,47)]
WAIT 1000
WRITE [(u1,10)(z3,9)(x1,49)(m9,68)]
BACKUP
DELETE [u5]
READ [u7]
SHOW
WRITE [(b8,21)(z4,69)(t3,29)(f2,67)(d0,88)]
READ [i0]
WRITE [(x0,38)(z7,30)(u2,70)]
WRITE [(h5,86)(h6,70)(q0,69)(c7,99)(m0,36)]
WRITE [(q7,20)(j1,68)(l2,95)(q1,3)(f0,61)]
WRITE [(v0,10)]
WRITE [(y8,93)(g9,94)(p9,71)]
WRITE [(e8,96)(j8,93)(j0,63)(t6,12)(c3,35)]
SHOW
READ [g3]
WRITE [(g7,57)]
DELETE [q0,l3,x1,e8]
WRITE [(w6,32)(k7,21)(w1,6)]
WRITE [(i1,7)(u0,87)(n7,86)]
