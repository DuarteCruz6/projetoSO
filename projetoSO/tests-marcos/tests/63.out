(a9, 32)
(g6, 10)
(i1, 10)
(k7, 23)
(o1, 2)
(u8, 32)
(z5, 34)
[(i8,74)]
(a1, 86)
(a2, 33)
(b5, 71)
(g3, 74)
(i0, 67)
(i5, 1)
(i8, 74)
(l1, 84)
(l9, 81)
(l3, 9)
(l5, 54)
(o9, 37)
(p4, 91)
(r2, 50)
(r9, 28)
(u9, 4)
(u5, 42)
(u7, 38)
(w8, 93)
(w5, 87)
(y9, 84)
(y1, 5)
[(b5,71)]
[(u7,38)]
(a1, 86)
(a2, 33)
(b5, 71)
(g3, 74)
(i0, 67)
(i5, 1)
(i8, 74)
(k0, 47)
(l1, 84)
(l9, 81)
(l3, 9)
(l5, 54)
(m9, 68)
(o9, 37)
(p4, 91)
(r2, 50)
(r9, 28)
(u1, 10)
(u9, 4)
(u7, 38)
(w8, 93)
(w5, 87)
(x1, 49)
(y9, 84)
(y1, 5)
(z3, 9)
[(i0,67)]
(a1, 86)
(a2, 33)
(b8, 21)
(b5, 71)
(c3, 35)
(c7, 99)
(d0, 88)
(e8, 96)
(f0, 61)
(f2, 67)
(g9, 94)
(g3, 74)
(h6, 70)
(h5, 86)
(i0, 67)
(i5, 1)
(i8, 74)
(j0, 63)
(j8, 93)
(j1, 68)
(k0, 47)
(l2, 95)
(l1, 84)
(l9, 81)
(l3, 9)
(l5, 54)
(m0, 36)
(m9, 68)
(o9, 37)
(p9, 71)
(p4, 91)
(q1, 3)
(q7, 20)
(q0, 69)
(r2, 50)
(r9, 28)
(t6, 12)
(t3, 29)
(u2, 70)
(u1, 10)
(u9, 4)
(u7, 38)
(v0, 10)
(w8, 93)
(w5, 87)
(x0, 38)
(x1, 49)
(y8, 93)
(y9, 84)
(y1, 5)
(z7, 30)
(z4, 69)
(z3, 9)
[(g3,74)]
